crates/context/config/src/client/env/proxy/query/active_proposals.rs

```diff
@@ -65,7 +65,7 @@ impl Method<Icp> for ActiveProposalRequest {
     }
 
     fn decode(response: Vec<u8>) -> eyre::Result<Self::Returns> {
-        let value = Decode!(&response, Self::Returns)?;
-        Ok(value)
+        let value = Decode!(&response, u32)?;
```

Comment:
@miraclx its okay for now as we are not hitting limit for u32 -> u16 conversion ?

Reply:
but why is it `u16` anyway, if all the contracts return `u32`?